app/src/main/java/com/example/recipeapp/fragments/ProfileFragment.java

```diff
@@ -65,22 +65,22 @@ public ProfileFragment() {
 
     @Override
     public View onCreateView(final @NonNull LayoutInflater inflater,
-                             @NonNull final ViewGroup container,
+                             @Nullable final ViewGroup container,
                              @Nullable final Bundle savedInstanceState) {
```

Comment:
Inconsistency of ordering, try doing a reformat CMD + Option + L and see if that clears it up.